book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
[ Optional Changes ]
The previous reviewer did not spot this but I highly recommend you make the change 👍🏽

- [ ] Getters and Setters for @rentals should be present. Kindly modify to `attr_accessor :title, :author, :rentals`

Reply:
Thanks for the information. I will also create the seet for `rentals`